npmpackagejsonlint.config.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+	// See https://www.npmjs.com/package/@wordpress/npm-package-json-lint-config
+	extends: '@wordpress/npm-package-json-lint-config',
+	rules: {
+		'prefer-no-devDependencies': 'error',
+		'prefer-property-order': 'off',
+		'require-author': 'error',
+		'require-bugs': 'off',
+		'require-homepage': 'off',
+		'require-keywords': 'off',
+		'require-license': 'error',
+		'require-name': 'error',
+		'require-version': 'error',
+		'valid-values-author': [ 'error', [ 'Automattic Inc.' ] ],
+		'valid-values-license': [ 'error', [ 'GPL-2.0-or-later' ] ],
```

Comment:
This one is also present in the shared config, although you might want to be more opinionated.

Reply:
Removed this and other redundant ones. https://github.com/Automattic/wp-calypso/pull/36534/commits/9db72d1c7ae01192e004c799a390cf96642cb7bd